modules/imaging_browser/jsx/ImagingBrowserDDT.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Created by dblader on 7/25/17.
+ */
+/* global DynamicDataTable */
+class ImagingBrowserDDT extends DynamicDataTable {
```

Comment:
What are the problems with my original suggestion of having an optional onSort prop as a callback even which triggers the post?